docs/whitepaper/federated_sidetree.md

```diff
@@ -194,22 +178,75 @@ It is necessary for a logical Hub to have tighter coupling than the interconnect
+
+When a batch writer accepts an operation for a unique suffix, it MUST also obtain a proof from the Merkle Tree log. This proof is written into the CAS along with the anchor. If the batch writer is unwilling to obtain proofs from a particular log, operations for unique suffixes checkpointed to that log MUST not be accepted.
+
+The DID controller should also be able to discover the acceptable logs that the batch writer accepts, along with the recommended (or default) log.
+
+Considerations:
+
+* Operations are ignored for suffixes where a required log proof was not provided.
+* TBD: Handling of multiple tree logs specified per unique suffix.
+
 ## DID strings and resolution
 
-The canonical representation for a DID string is:
+The DID string enables the resolver to determine:
+
+* The unique suffix of the DID - the chained operations are applied sequentially to obtain a DID document.
+* Merkle Tree logs that enable the resolver to determine if it has knowledge of the latest chained operations for the unique suffix. 
+* Discovery information to help determine endpoints of the operators or hubs.
+
+The canonical representation for an Orbital DID string is:
 
-did:<method name>:<tree version><tree id>:<unique suffix>
+`did:orb:<checkpoint cid>:<unique suffix>`
 
-* Method Name: The DID method defined in this document (currently did:multitree).
-* Unique Suffix: The unique suffix is defined by the Sidetree protocol specification.
-* Tree version (/type): Enables signaling metadata about the Tree for Multitree processors. This field allows for potential system evolution (future-proofing), similar to [IPFS CIDs](https://docs.ipfs.io/concepts/content-addressing/) or [multiformats](https://multiformats.io).
-* Tree ID: To mitigate against rollbacks, a tree identifier can be provided such that the resolution MUST fail if unique suffix chain does not include the tree. For greater clarity, this does not mean that the DID suffix is locked to that tree – the controller may continue the unique suffix chain onto another tree following the rules described above.
-  * Note: the tree id needs more description based on the concrete implementation. The TrustBloc implementation currently includes a genesis configuration hash in this position. The Tree ID MUST be unique based on some verifiable genesis mechanism.
-  * As mentioned earlier, a more explicit operation to move DIDs between trees is useful. In this case, there is some concern if the original tree disappears but the DID is still in use. An explicit operation could act as a checkpoint of the old tree identifier onto the new tree.
+The checkpoint is the CID of the anchor that contains a checkpoint for the unique suffix. The resolver ensure that it using an anchor graph, for the unique suffix, that includes the checkpoint CID. The Merkle Tree log associated to the unique suffix is obtained from the latest checkpoint in the graph rooted by the checkpoint CID.
```

Comment:
This implies that there is always checkpoint operation - is there a case for DID resolution such that DID was not imported from some tree (so no checkpoint operation)?

Reply:
We should probably always explicitly checkpoint prior to a resolution to set the tree. We could also consider a combined create+checkpoint operation.